roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/ruby
+username = "admin"
+password = "changeme"
```

Comment:
It is better to add a comment saying that the clone playbook will change the credentials to admin/changeme.  If the user had changed the password after cloning, before running this script - then these credentials need to be updated accordingly.